src/EditorFeatures/CSharpTest/ChangeSignature/ChangeSignature_Delegates.cs

```diff
@@ -19,7 +19,7 @@ public partial class ChangeSignatureTests : AbstractChangeSignatureTests
         protected override CodeRefactoringProvider CreateCodeRefactoringProvider(Workspace workspace, TestParameters parameters)
             => new ChangeSignatureCodeRefactoringProvider();
 
-        protected override string GetLanguage()
+        protected internal override string GetLanguage()
```

Comment:
?

Reply:
This will go away once the tests are converted to the new test framework.

Reply:
WFM.